diff --git a/chrome/browser/themes/theme_service_factory.cc b/chrome/browser/themes/theme_service_factory.cc
index 783fd6aa20dacf1c85408d4986147ce3578690d5..e13971bc042d65549b8b402e6d01acb96927cf5b 100644
--- a/chrome/browser/themes/theme_service_factory.cc
+++ b/chrome/browser/themes/theme_service_factory.cc
@@ -103,6 +103,7 @@ ThemeServiceFactory::~ThemeServiceFactory() = default;
 
 KeyedService* ThemeServiceFactory::BuildServiceInstanceFor(
     content::BrowserContext* profile) const {
+  BRAVE_THEMESERVICEFACTORY_BUILDSERVICEINSTANCEFOR
 #if BUILDFLAG(IS_LINUX)
   using ThemeService = ThemeServiceAuraLinux;
 #endif
