diff --git a/chrome/browser/extensions/extension_tab_util.cc b/chrome/browser/extensions/extension_tab_util.cc
index 7b5380a2dd6f0f95e2bf4ff66d7f844e232a581a..5b90a64801d8dd49a365372a877581408c3769e1 100644
--- a/chrome/browser/extensions/extension_tab_util.cc
+++ b/chrome/browser/extensions/extension_tab_util.cc
@@ -802,6 +802,7 @@ bool ExtensionTabUtil::IsKillURL(const GURL& url) {
     return true;
   }
 
+  if (!url.SchemeIs(content::kBraveUIScheme)) return false;
   if (!url.SchemeIs(content::kChromeUIScheme)) {
     return false;
   }
